src/evict.c

```diff
@@ -738,12 +738,18 @@ int performEvictions(void) {
         /* At this point, we have run out of evictable items.  It's possible
          * that some items are being freed in the lazyfree thread.  Perform a
          * short wait here if such jobs exist, but don't wait long.  */
-        if (bioPendingJobsOfType(BIO_LAZY_FREE)) {
-            usleep(eviction_time_limit_us);
+        mstime_t lazyfree_latency;
+        latencyStartMonitor(lazyfree_latency);
+        while(bioPendingJobsOfType(BIO_LAZY_FREE) &&
```

Comment:
```suggestion
        while (bioPendingJobsOfType(BIO_LAZY_FREE) &&
```

Reply:
i prefer the one with the space, but for the record, this one is really inconsistent in redis.
there are about the same number of `while` statement with space as there are without space.
@soloestoy your call...

Reply:
OK, let's unify the while style in evict.c first.